examples/big-bang/README.md

```diff
@@ -42,3 +42,125 @@ To view the example in its entirety, select the `Edit this page` link below the
 :::
 
 <ExampleYAML example="big-bang/yolo" showLink={false} />
+
+## Big Bang Scrape Zarf Agent Metrics
+
+The Zarf Agent emits Prometheus metrics that can be scraped by Big Bang's Prometheus Operator. To enable this, set `monitoring.enabled` to true in the `config/disable-all.yaml`, and uncomment the `disable-all.yaml` under the `components.extentions.bigbang.valuesFiles` section in `zarf.yaml`.
+
+Finally, create a `ServiceMonitor` for the Zarf Agent. Since this the Zarf Agent exposes an `https` port, we need to provide the `bearerTokenFile` and `tlsConfig` to specify the TLS settings for scraping agaist the service. 
+
+```yaml
+kubectl create -f -<<EOF
```

Comment:
We wrap `kubectl` under `zarf tools` and could re-plug that here.  I also wonder if this is worth being a sub-example like the YOLO mode package is where we show the Zarf YAML that would apply this instead of the raw service monitor with a `kubectl apply`

Reply:
sounds like a good idea to me

Reply:
✅

Reply:
hey @Racer159 , I was diving into the YOLO mode example and I don't know if we can give an example in this scenario as the `agent-hook` will not be around to scrape. I could be missing something, though. Let me know your thoughts around this, thanks! 🦄

Reply:
Suggestion's from @Racer159 after 1on1:
- [ ] update the bundle to install in monitoring ns
- [ ] refactor that zarf package now understanding that array and potential for an array of manifests should it be needed taking out the cmd
- [ ] simplify the port-forwarding piece by adding labels
consolidate the demos so that there is not 2 different demos - pointing the the scrape - [ ] Zarf Agent demo from the big bang demo 
- [ ] get a test in around prometheus running
- [ ] take out hello zarf since it does not have much to do with scraping zarf agent

Reply:
On num 3 this might be able to be achieved by adding a secondary service that is not controlled by the operator

Reply:
great idea